CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 - Add the new `go.opentelemetry.io/contrib/instrgen` package to provide auto-generated source code instrumentation. (#3068, #3108)
 
+### Fixed
+
+- config: Support Console exporter for LoggerProvider (#5432, #5850)
```

Comment:
```suggestion
- Support for stdoutlog exporter in `go.opentelemetry.io/contrib/config`. (#5850)
```

This also isn't a bugfix, but a new feature. It should be in the `Added` section.